package/src/components/Message/MessageSimple/MessageSimple.tsx

```diff
@@ -147,18 +147,17 @@ const areEqual = <StreamChatGenerics extends DefaultStreamChatGenerics = Default
   const prevMessageAttachments = prevMessage.attachments;
   const nextMessageAttachments = nextMessage.attachments;
   const attachmentsEqual =
-    Array.isArray(prevMessageAttachments) &&
-    Array.isArray(nextMessageAttachments) &&
-    prevMessageAttachments.length === nextMessageAttachments.length &&
-    prevMessageAttachments.every((attachment, index) => {
-      const attachmentKeysEqual =
-        attachment.type === 'image'
-          ? attachment.image_url === nextMessageAttachments[index].image_url &&
-            attachment.thumb_url === nextMessageAttachments[index].thumb_url
-          : attachment.type === nextMessageAttachments[index].type;
-
-      return attachmentKeysEqual;
-    });
+    Array.isArray(prevMessageAttachments) && Array.isArray(nextMessageAttachments)
+      ? prevMessageAttachments.length === nextMessageAttachments.length &&
+        prevMessageAttachments.every((attachment, index) => {
+          const attachmentKeysEqual =
+            attachment.image_url === nextMessageAttachments[index].image_url &&
+            attachment.og_scrape_url === nextMessageAttachments[index].og_scrape_url &&
+            attachment.thumb_url === nextMessageAttachments[index].thumb_url;
+
+          return attachmentKeysEqual;
```

Comment:
We don't need `isAttachmentEqual` check here?

Reply:
Nope, I think we have discussed it in the call before.

Reply:
Yup, since we are not using attachment inside MessageSimple directly. I think we can remove this entire check ... but leave it for now!! Lets deal with it separately

Reply:
So this is just the refactoring of the initial code, nothing else. I Simply used, `attachmentKeysEqual` to store the condition and returned it.